opentype/api/font/font.go

```diff
@@ -229,22 +229,22 @@ var bhedTag = loader.MustNewTag("bhed")
 
 // LoadHeadTable loads the table corresponding to the 'head' tag.
 // If a 'bhed' Apple table is present, it replaces the 'head' one.
-func LoadHeadTable(ld *loader.Loader) (tables.Head, error) {
-	var (
-		s   []byte
-		err error
-	)
+//
+// 'buffer' may be provided to reduce allocations; the return Head is guaranteed
+// not to retain any reference on 'buffer'
+func LoadHeadTable(ld *loader.Loader, buffer []byte) (tables.Head, []byte, error) {
```

Comment:
Should we be explicit for this public API that `nil` will cause a full allocation, but also that if the buffer is too small a new one will be allocated?

Reply:
Fair enough, thanks. I've added some documentation.